lib/dal/src/layer_db_types/content_types.rs

```diff
@@ -258,6 +260,18 @@ pub struct InputSocketContentV1 {
     pub connection_annotations: Vec<ConnectionAnnotation>,
 }
 
+#[derive(Debug, Clone, EnumDiscriminants, Serialize, Deserialize, PartialEq)]
+pub enum ModuleContent {
+    V1(ModuleContentV1),
+}
+
+#[derive(Debug, Clone, Deserialize, Serialize, PartialEq)]
+pub struct ModuleContentV1 {
```

Comment:
do we also need: 
`display_name` `name` etc?

Reply:
That's not part of the module - that's part of the Schema / Schema Variant. The module just has a name and a root_hash.

Reply:
`export interface LocalModuleSummary {
  name: string;
  hash: ModuleHash;
  isBuiltin: boolean;
}

export interface LocalModuleDetails {
  name: string;
  version: string;
  description: string;
  createdAt: IsoDateString;
  createdBy: string;
  schemas: string[];
  funcs: PkgFuncView[];
  hash: ModuleHash;
  kind: "module" | "workspaceExport";
}`
I think there's a name/description/etc. for that too - here's from app/web

Reply:
I wonder if modules implicitly only have one schema (not saying how it should always be, just how it is now) which is influencing what's returned in that route?

Reply:
I don't think so.. even now we have multiple schemas in a module